src/components/admin/app-navbar/AppNavbar.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+
+  <vuestic-navbar>
+    <header-selector slot="selector" v-model="valueProxy"/>
+    <span slot="logo" class="i-vuestic"></span>
+    <span slot="center">
+      {{$t('navbar.messageUs')}}&nbsp;<a href="mailto:hello@epicmax.co">hello@epicmax.co</a>
+    </span>
+    <message-dropdown :options="this.messages"></message-dropdown>
+    <notification-dropdown :options="this.notifications"></notification-dropdown>
+    <language-dropdown :options="this.languages"></language-dropdown>
+    <profile-dropdown :options="this.profiles">
+      <img src="http://i.imgur.com/nfa5itq.png"/>
+    </profile-dropdown>
+  </vuestic-navbar>
+
+</template>
+
+<script>
+  import VuesticNavbar from '../../../vuestic-theme/vuestic-components/vuestic-navbar/VuesticNavbar'
+  import HeaderSelector from './components/HeaderSelector'
+  import LanguageDropdown from './components/dropdowns/LanguageDropdown'
+  import ProfileDropdown from './components/dropdowns/ProfileDropdown'
+  import NotificationDropdown from './components/dropdowns/NotificationDropdown'
+  import MessageDropdown from './components/dropdowns/MessageDropdown'
+
+  export default {
+    name: 'app-navbar',
+
+    components: {
+      VuesticNavbar,
+      HeaderSelector,
+      MessageDropdown,
+      NotificationDropdown,
+      LanguageDropdown,
+      ProfileDropdown
+    },
+
+    props: {
+      isOpen: {
+        type: Boolean,
+        required: true
+      }
+    },
+
+    data () {
```

Comment:
More correct solution would be to use computed instead of data. This data is not meant to be reactive.